go/common/headers.go

```diff
@@ -23,24 +23,26 @@ var hasherPool = sync.Pool{
 // BatchHeader is a public / plaintext struct that holds common properties of batches.
 // Making changes to this struct will require GRPC + GRPC Converters regen
 type BatchHeader struct {
-	ParentHash       L2BatchHash `json:"parentHash"`
-	Root             StateRoot   `json:"stateRoot"`
-	TxHash           common.Hash `json:"transactionsRoot"`
-	ReceiptHash      common.Hash `json:"receiptsRoot"`
-	Number           *big.Int    `json:"number"`           // height of the batch
-	SequencerOrderNo *big.Int    `json:"sequencerOrderNo"` // multiple batches can be created with the same height in case of L1 reorgs. The sequencer is responsible for including all of them in the rollups.
-	GasLimit         uint64      `json:"gasLimit"`
-	GasUsed          uint64      `json:"gasUsed"`
-	Time             uint64      `json:"timestamp"`
-	Extra            []byte      `json:"extraData"`
-	BaseFee          *big.Int    `json:"baseFee"`
+	ParentHash       L2BatchHash    `json:"parentHash"`
+	Root             StateRoot      `json:"stateRoot"`
+	TxHash           common.Hash    `json:"transactionsRoot"`
+	ReceiptHash      common.Hash    `json:"receiptsRoot"`
+	Number           *big.Int       `json:"number"`           // height of the batch
+	SequencerOrderNo *big.Int       `json:"sequencerOrderNo"` // multiple batches can be created with the same height in case of L1 reorgs. The sequencer is responsible for including all of them in the rollups.
+	GasLimit         uint64         `json:"gasLimit"`
```

Comment:
Our maximum gas limit per batch should be the one from ethereum/12.
our batch time =1s